_sass/code.scss

```diff
@@ -48,17 +50,61 @@ a:visited code {
 
 // ```[LANG]...```
 div.highlighter-rouge,
-div.listingblock > div.content {
-  padding: $sp-3;
+div.listingblock > div.content,
+figure.highlight {
   margin-top: 0;
   margin-bottom: $sp-3;
-  overflow-x: auto;
   background-color: $code-background-color;
   border-radius: $border-radius;
   box-shadow: none;
   -webkit-overflow-scrolling: touch;
+  position: relative;
+  padding: 0;
+
+  > button {
+    width: $sp-3;
+    height: $sp-3;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    right: 0;
+    border: $sp-3 solid $code-background-color;
+    background-color: $code-background-color;
+    outline: none;
```

Comment:
```suggestion
    outline: none;
```

Is there a reason why this needs to be here? If we remove this, then the button has a built-in browser focus state; see this in chrome:

<img width="768" alt="Screenshot 2022-12-26 at 2 31 15 PM" src="https://user-images.githubusercontent.com/14893287/209586595-7e05730c-e397-4c72-b92a-07c8912e69e2.png">